ui/app/AppLayouts/Chat/panels/communities/PermissionQualificationPanel.qml

```diff
@@ -0,0 +1,56 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+Control {
+    property int qualifyingAddresses: 1
+    property int knownAddresses: 1
+    property int unknownAddresses: 1
+
+    verticalPadding: 16
+    horizontalPadding: 18
+
+    background: Rectangle {
+        border.color: Style.current.border
+        radius: Style.current.radius
+        color: Style.current.transparent
+    }
+
+    contentItem: RowLayout {
+        spacing: 4
+
+        StatusIcon {
+            Layout.preferredWidth: 22
+            Layout.preferredHeight: 22
+
+            Layout.alignment: Qt.AlignTop
+
+            color: Theme.palette.baseColor1
+            icon: "communities"
+        }
+
+        StatusBaseText {
+            Layout.fillWidth: true
+
+            wrapMode: Text.Wrap
+
+            font.pixelSize: 15
```

Comment:
I guess it should be Style.current.primaryTextFontSize, right? This will also enable the font size increase/decrease from settings.